isisd/isis_nb_config.c

```diff
@@ -1866,9 +1870,8 @@ int lib_interface_isis_create(struct nb_cb_create_args *args)
 		}
 		break;
 	case NB_EV_APPLY:
-		ifp = nb_running_get_entry(args->dnode, NULL, true);
-		if (ifp)
-			area = isis_area_lookup(area_tag, ifp->vrf_id);
+		if (vrf_name != NULL)
```

Comment:
Unnecessary null check since the VRF leaf has a default value and thus always exist.

Reply:
This is resolved.